graviti/portex/base.py

```diff
@@ -18,6 +29,36 @@
 
 _INDENT = " " * 2
 
+_T = TypeVar("_T", bound="PortexType")
+
+PYARROW_TYPE_ID_TO_PORTEX_TYPE = {}
+
+
+class RegisterPortexType:
```

Comment:
will it be better if making `PYARROW_TYPE_ID_TO_PORTEX_TYPE ` and `_get_pyarrow_type_id` as the member of `RegisterPortexType `

Reply:
Also the name `RegisterPortexType ` seems unrelated with pyarrow?